src/pages/index.tsx

```diff
@@ -162,6 +173,17 @@ const IndexPage = ({ data }) => {
           </CtaLink>
         </Grid>
       </Section>
+      <div className="su-h-400 lg:su-h-600 su-overflow-hidden su-relative">
+        <Parallax offset={30}>
+          <div style={{ backgroundImage: 'url(https://www.space.com/images/i/000/082/219/original/VLT-Pano-MCloudsMilkyWay_6068-net.jpg?interpolation=lanczos-none&fit=around|1024:1024' }} className="su--mt-100 su-h-800 su-flex su-items-center su-justify-center su-bg-center su-bg-cover" />
+          <div className="su-absolute su-top-0 su-left-0 su-w-full su-h-full su-content-[''] su-block su-z-1 su-bg-black/40" />
+        </Parallax>
+        <div className="su-absolute su-top-[50%] su-left-[50%] su-translate-x-[-50%] su-translate-y-[-50%] su-text-shadow-lg su-z-10 su-max-w-800">
+          <Parallax offset={70}>
+            <Text leading="none" align="center" font="druk-wide" size={7} color="white" weight="bold" className="">To infinity and beyond</Text>
+          </Parallax>
+        </div>
+      </div>
```

Comment:
In this example where the foreground text move more, I just set the parallax `offset` to a larger value for the text.